### PR TITLE
numpy instance method floordiv

### LINKED ISSUE #7737: __floordiv__

### PR DESCRIPTION
**Pull request by abhi27-web**
Close #7737

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 1]
```diff
@@ -1272,6 +1272,43 @@ def test_numpy_instance_truediv__(
         frontend=frontend,
     )
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1274]: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1275:1: E302 expected 2 blank lines, found 1

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
I found a small formatting error as described in the comment below. Now just wait for the checks to complete to see if there is anything else wrong.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py (hunk 1 above)
Comment (abhi27-web) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1274]: Made the change!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 2]
```diff
@@ -1273,6 +1273,44 @@ def test_numpy_instance_truediv__(
...
 
 
+@handle_frontend_method(
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1276]: E   TypeError: handle_frontend_method() missing 3 required keyword-only arguments: 'class_tree', 'init_tree', and 'method_name'

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
I found something wrong with how you generate data for your test, check the comment below. Thank you.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py (hunk 2 above)
Comment (abhi27-web) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1276]: Made the changes

**Comment by abhi27-web** [comment]
@Daniel4078 Can you help me with this error.

**Comment by Daniel4078** [comment]
> @Daniel4078 Can you help me with this error.
What error do you mean by "this error"?

**Comment by Daniel4078** [comment]
> > > @Daniel4078 Can you help me with this error.
> > > What error do you mean by "this error"?
> 
> 1 test is failing and I am unable to understand the reason behind it!

check this out, this is posted in the internal channel of ivy's discord.
Hello @everyone,

Just pushed an update of how the test flags are handled, the inference of with_out keyword argument was removed to reduce unneeded performance overhead. 

Some tests will start to fail, i'm working on fixing most of them now but if you are working on a implementing a test and it starts to fail due to unexpected keyword argument 'out' simply remove the with_out keyword and hardcode the test_value to False as done here: https://github.com/unifyai/ivy/commit/fd7ee2daec482b6a56e06b52b395c2bf2777c1ff

Also, some updates will be made in the next few days to how the test flags are passed around, will make things more cleaner and readable. Hopefully no more test factoring after this 😂 , Thanks!

**Comment by abhi27-web** [comment]
> > > > @Daniel4078 Can you help me with this error.
> > > > What error do you mean by "this error"?
> > 
> > 
> > 1 test is failing and I am unable to understand the reason behind it!
> 
> check this out, this is posted in the internal channel of ivy's discord. Hello @everyone,
> 
> Just pushed an update of how the test flags are handled, the inference of with_out keyword argument was removed to reduce unneeded performance overhead.
> 
> Some tests will start to fail, i'm working on fixing most of them now but if you are working on a implementing a test and it starts to fail due to unexpected keyword argument 'out' simply remove the with_out keyword and hardcode the test_value to False as done here: [fd7ee2d](https://github.com/unifyai/ivy/commit/fd7ee2daec482b6a56e06b52b395c2bf2777c1ff)
> 
> Also, some updates will be made in the next few days to how the test flags are passed around, will make things more cleaner and readable. Hopefully no more test factoring after this 😂 , Thanks!

But I have not used out keyword in my code.

**Comment by Daniel4078** [comment]
Understand, let me have a closer look.

**Comment by Daniel4078** [comment]
I think the example you should look at is this:
def test_numpy_instance_sub__(
    dtype_and_x,
    as_variable: pf.AsVariableFlags,
    native_array: pf.NativeArrayFlags,
    init_num_positional_args: pf.NumPositionalArgFn,
    method_num_positional_args: pf.NumPositionalArgMethod,
    frontend_method_data,
    frontend,
):

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**